content/transfer-photos-from-phone.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Transferring photos from your phone to Pop!_OS
+description: >
+  How to move your photos from your Android or iPhone to Pop!_OS
+keywords:
+  - System76
+  - Apple
+  - OS X
+  - macOS
+  - Android
+  - Google
+  - switch
+  - PopOS
+  - Pop!\_OS
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: switching
+tableOfContents: true
+---
+
+## From Pixel/Android to Pop!_OS
+
+### Changing the USB Preference
+
+When you connect your Pixel/Android device to your system there is a notification about USB Preferences which can be reached when you swipe from the top of the screen like the screenshot below:
+
+![Pixel USB Notification](/images/transfer-photos-from-phone/Pixel-USB-Notification.png)
+
+Once you press on the notification the follow screen will appear:
+
+![Pixel USB Preferences](/images/transfer-photos-from-phone/Pixel-USB-Preferences-1.png)
+
+Simply change "no data transfer" to "File transfer / Android Auto":
+
+![Pixel USB Preferences](/images/transfer-photos-from-phone/Pixel-USB-Preferences-2.png)
+
+### Open the Files application
+
+Next open the <u>Files</u> application and right-click on the Pixel 8 item (in this example it may say the OEM of your device such as Samsung) in the menu on the left side of the window to access the file system on the device.
```

Comment:
>right-click on the Pixel 8 item (in this example it may say the OEM of your device such as Samsung) in the menu on the left side of the window to access the file system on the device.

I feel like this could be much cleaner and less Pixel 8 specific. Something along the lines of, "The name of your device should appear in the left column of the file browser. Click on it to access the device's files."

Also you said right-click, when I think a normal/left click is probably what you meant.